react-function-components/src/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton(props) {
+  return (
+    <div>
```

Comment:
The `<div>` here is unnecessary, we don't want to randomly add extra elements if we don't have to. This will also make it so you couldn;t have multiple buttons setting next to each other on the same line.